static-assets/components/cstudio-common/common-api.js

```diff
@@ -3079,10 +3079,10 @@ var nodeOpen = false,
             getPermissionsServiceUrl: "/api/1/services/api/1/security/get-user-permissions.json",
             lookupAuthoringRoleServiceUrl : "/api/1/services/api/1/security/get-user-roles.json",
             verifyAuthTicketUrl: "/api/1/services/api/1/user/validate-token.json",
-            getUserInfoServiceURL: "/api/2/user",
+            getUserInfoServiceURL: "/api/2/users",
```

Comment:
Seems to be missing `/me`

Reply:
It can be 'me' or user id:

```
getUserInfo: function(callback, user) {
                var serviceUrl = this.getUserInfoServiceURL;
                var user = !user ? 'me' : user;
```